### PR TITLE
Fix beta pill label breaking

### PR DESCRIPTION
**Pull request by luixxiul**
Closes https://github.com/vector-im/element-web/issues/21566

This commit ensures word break is not used for beta pill label (`ベータ版`) in CJK languages.

after

Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

type: defect

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Fix beta pill label breaking ([\#8162](https://github.com/matrix-org/matrix-react-sdk/pull/8162)). Fixes vector-im/element-web#21566. Contributed by @luixxiul.

Preview: https://pr8162--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: res/css/views/beta/_BetaCard.scss [hunk 1]
```diff
@@ -101,9 +101,10 @@ limitations under the License.
     font-size: 12px;
     font-weight: $font-semi-bold;
     line-height: 15px;
-    color: #FFFFFF;
+    color: #fff;
```
Comment (t3chguy) [inline res/css/views/beta/_BetaCard.scss:104]: this is an unrelated change

Comment (luixxiul) [reply res/css/views/beta/_BetaCard.scss:104]: Yes, I'll revert this line.

**Review by t3chguy — APPROVED** [review]
This looks sane, thanks!
